addons/website_mass_mailing/views/mass_mailing_view.xml

```diff
@@ -6,19 +6,19 @@
         <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_list_form"/>
         <field name="arch" type="xml">
             <xpath expr="//sheet" position="inside">
-                <notebook groups="website_mass_mailing.group_website_popup_on_exit">
-                    <page string="Popup Content">
-                        <field name="popup_content" widget="html" options="{
-                                'snippets': 'website.snippets',
-                                'cssEdit': 'website_mass_mailing.iframe_css_assets_edit',
-                                'wrapper': 'website_mass_mailing.edition.wrapper',
-                            }"/>
+                <notebook>
+                    <page string="Website popups">
+                        <field name="website_popup_ids">
+                            <tree editable="bottom">
+                                <field name="website_id"/>
+                            </tree>
+                        </field>
                     </page>
                 </notebook>
             </xpath>
             <xpath expr="//div[hasclass('oe_title')]" position="inside">
-                <group groups="website_mass_mailing.group_website_popup_on_exit">
-                    <field name="popup_redirect_url"/>
+                <group>
+                    <field name="toast_content"/>
```

Comment:
You are positioning a group with field inside a div which has the oe_title class... seems strange to me :)